django_dbconn_retry/__init__.py

```diff
@@ -39,6 +40,8 @@
 else:
     _operror_types += (MySQLdb.OperationalError,)
 
+MAX_DBCONN_RETRY_TIMES = getattr(settings, "MAX_DBCONN_RETRY_TIMES", 1)
```

Comment:
You might want to consider getting the setting inside the method. That makes testing easier.